src/pages/components/text-input/usage.mdx

```diff
@@ -132,99 +152,365 @@ reflect the length of the content you expect the user to enter.
+
+![Example showing optional field indicator](./images/text-input-usage-optional-do.png)
 
-Real time validation helps to streamline the process and keep data clean when
-the user is filling out forms. For full guidelines, refer to the
-[forms](/components/form/usage) usage page.
+</DoDont>
+<DoDont
+colMd={4} colLg={4}
+caption="If the majority of the fields are optional, mark only the required field labels with (required).">
+
+![Example showing required field indicator](./images/text-input-usage-required-do.png)
+
+</DoDont>
+</DoDontRow>
+
+### Interactions
+
+#### Mouse
+
+Users can activate a text input by clicking on the field container. A separate
+click is required to activate any additional actions associated with the text
+input such as a tooltip or password visibility toggle.
 
 <Row>
 <Column colLg={8}>
 
-![Validation text example](images/text-input-usage-2.gif)
+![Example showing click target areas for text input](images/text-input-usage-mouse.png)
 
 </Column>
 </Row>
 
-### Character count
+#### Keyboard
+
+For additional keyboard interactions, see the
+[accessibility tab](/components/text-input/accessibility).
+
+| Key                                                            | Interaction                                                                            |
+| -------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| _Tab_                                                          | Brings focus to the text input.                                                        |
+| _Enter_ or _Space_                                             | Opens any associated actions added to the input, such as a password visibility toggle. |
+| _Esc_                                                          | Closes any associated actions added to the input.                                      |
+| _ctrl_ + _left/right arrows_ <br/> _opt_ + _left/right arrows_ | Moves you word by word inside the field.                                               |
+| _ctrl_ + _up/down arrow_ <br/> _opt_ +_up/down arrows_         | Relocates you to the start or end of the input content.                                |
 
-Text fields and text areas may have a character limit and counter. Use the
-character limit and counter to let users know how long their entry can be before
-they begin typing.
+## Text input
 
-While useful for data collection, character count is not an intuitive way to
-communicate content length to users. Character count allows a user to understand
-an input error if they surpass the character limit.
+Use a text input when the expected user input is a single line of text. Text
+inputs have a fixed height and are used a simple free-form data entry. Users can
+enter any combination of letters, numbers, or symbols.
 
 <Row>
 <Column colLg={8}>
 
-![An example of a text area.](images/text-area-usage-1.png)
```

Comment:
- This images needs side borders, too.

<img width="783" alt="Screen Shot 2022-08-26 at 3 17 42 PM" src="https://user-images.githubusercontent.com/43969356/186976234-ca11d1fe-ee9e-4c92-bf31-551b38eba119.png">
.